README.md

```diff
@@ -42,20 +42,20 @@ On all platforms run the following in the VireoSDK folder to install JavaScript-
 - emsdk toolchain (see following)
 
 ### Installing the emsdk toolchain
-The emsdk toolchain is used to cross-compile Vireo to Vireo.js to run in Node.js and Browser environments.
+emscripten (emsdk) is used to cross-compile Vireo to Vireo.js to run in Node.js and web browser environments.
```

Comment:
Emscripten is a set of command line tools used to cross-compile...  The emsdk tool is used to install binaries of the emscripten command line tools and its required dependencies.

Reply:
You're right. I reverted this.